### PR TITLE
README Formatting

### PR DESCRIPTION
**Pull request by ElectronicsArchiver**
Please check the **[Preview]**.

### Changes

- General formatting / spacing
- Added license, author, version badge
- Liked author, license

### Repository Suggestions

- Disable unused widgets:
  - `Packages`
  - `Releases`

- Disable unused tabs:
  - `Projects`
  - `Actions`
  - `Wiki` 

[Preview]: https://github.com/ElectronicsArchiver/reap

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+
+# REAP ![Badge Version] [![Badge License]][License] [![Badge Author]][Author]
+
+*A **Library** to provide easy access to* ***ProcFS***
```
Comment (nickeldan) [inline README.md:4]: If you revert this line to what it was before, I'll merge this PR.

Comment (ElectronicsArchiver) [reply README.md:4]: `this line`, line 2 or 4?
I'm guessing 4?

Comment (nickeldan) [reply README.md:4]: Yeah, 4.

Comment (ElectronicsArchiver) [reply README.md:4]: Well I took the content of the `Overview` section as the description,
and at the original position there was the table which is now two
badges.

So do you just want the original wording of the description?

Comment (nickeldan) [reply README.md:4]: Yeah, "The REAP library provides Ridiculously Easy Access to Procfs."

**Pull request merged**